src/averageAges.js

```diff
@@ -20,6 +20,12 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  const arr = [...people].filter(x => x.sex === 'm'
```

Comment:
why arr? let's name this variable `men`, cause it's exactly what you're searching for

Reply:
is filter a mutating method?

Reply:
```suggestion
  const arr = people.filter(x => x.sex === 'm'
```